ambry-api/src/main/java/com/github/ambry/config/ServerReplicationMode.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.config;
+public enum ServerReplicationMode {
```

Comment:
We should move these configs to storeConfig.

Reply:
Keeping this as a general purpose enum which can be used by other classes if needed.